### DISCUSSION TITLE
Adding archipelagos to the index

### DISCUSSION THREAD

**Discussion by Janjko** [osm-search/Nominatim#2899] [Ideas]
Would it be possible or wanted to add archipelagos to the Nominatim searchable index? I have a feeling those are pretty important features, and it would be nice to see them pop up in the search.

https://wiki.openstreetmap.org/wiki/Tag:place%3Darchipelago

**Comment by lonvia** [comment]
Can you given an example of an archipelago that isn't found?

**Reply by Janjko** [reply]
I've just found the archipelago. I guess I wrote it wrong. Sorry!
